charts/istio-alerts/templates/service-prometheusrule.yaml

```diff
@@ -85,5 +85,37 @@ spec:
         {{ toYaml . | nindent 8}}
         {{- end }}
     {{- end }}
-  {{- end }}
+    {{- end }}
+
+  - name: {{ .Release.Name }}.{{ .Release.Namespace }}.{{ .Chart.Name }}.alertRulesSelectorsValidity
+    rules:
+    {{- with .Values.serviceRules.destinationServiceSelectorValidity }}
+    - alert: DestinationServiceSelectorValidity
+      annotations:
+        summary: >-
+          DestinationServiceSelector for istio-alerts is invalid
+        runbook_url: {{ $values.defaults.runbookUrl}}#Alert-Rules-Selectors-Validity
+        description: >-
+          The DaemonsetSelector used for daemonset level alerts did not return any data.
```

Comment:
Copy-paste error... this message should be accurate to the alarm, it's about istio..

Reply:
resolved in new PR